src/net/sourceforge/kolmafia/session/EquipmentManager.java

```diff
@@ -927,8 +927,12 @@ public static final void addConditionalSkills(final Slot slot, AdventureResult i
         KoLCharacter.addAvailableSkill(SkillPool.SWOOP_LIKE_A_BAT);
         KoLCharacter.addAvailableSkill(SkillPool.SUMMON_CAULDRON_OF_BATS);
       }
-      case ItemPool.SHERIFF_BADGE, ItemPool.SHERIFF_PISTOL, ItemPool.SHERIFF_MOUSTACHE  -> {
-        KoLCharacter.addAvailableSkill(SkillPool.ASSERT_YOUR_AUTHORITY);
+      case ItemPool.SHERIFF_BADGE  -> {
```

Comment:
not quite -- this say "when I equip the badge, if the other two are equipped, add the skill", but does nothing e.g. when equipping the pistol. so you need two extra cases for the pistol and moustache.

I can do it if you'd like?

Reply:
you also need to run spotless for formatting

Reply:
Sure, I see what you mean but am engrossed in my day job now.